pkgs/development/libraries/vte/default.nix

```diff
@@ -78,6 +78,8 @@ stdenv.mkDerivation rec {
 
   mesonFlags = lib.optionals (!systemdSupport) [
     "-D_systemd=false"
+  ] ++ lib.optionals stdenv.isDarwin [
+    "-D_b_symbolic_functions=false"
```

Comment:
Can you add a comment why this is required?

Reply:
Done.